code/game/machinery/computer/buildandrepair.dm

```diff
@@ -46,15 +17,20 @@
 	var/powernet = null
 	var/list/records = null
 	var/contain_parts = 1
-	toolspeed = 1
-	usesound = 'sound/items/deconstruct.ogg'
 
 /obj/item/circuitboard/computer
 	board_type = "computer"
 
 /obj/item/circuitboard/machine
 	board_type = "machine"
 
+/obj/item/circuitboard/Initialize(mapload)
+	. = ..()
+	format_board_name()
+
+/obj/item/circuitboard/proc/format_board_name()
+	name = "[initial(name)] ([board_name])"
```

Comment:
`ASSERT()` won't work here I think since the `board_name` for `/obj/item/circuitboard` will always be null. So I'll do this instead to stop any formatting errors:
```dm
if(board_name)
	name = "[initial(name)] ([board_name])"
else
	name = "[initial(name)]"
```